airflow/bin/cli.py

```diff
@@ -1789,6 +1817,12 @@ class CLIFactory:
         'dag_regex': Arg(
             ("-dx", "--dag_regex"),
             "Search dag_id as regex instead of exact string", "store_true"),
+        # show_dag
+        'save': Arg(("-s", "--save"), "Saves the result to the indicated file."),
```

Comment:
Can you mention something about `a.dot` example here? And probably `a.png` too.

Reply:
I added examples.